libraries/Microsoft.Bot.Builder.Dialogs/Microsoft.Bot.Builder.Dialogs.csproj

```diff
@@ -18,7 +18,40 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Description>This library implements .NET Simple Dialog classes </Description>
+    <Description>
+      This library implements .NET Simple Dialog classes

```

Comment:
Is this line necessary? As this is the same line from the _Summary_

Reply:
I am not sure it is, but it was like that before so I don't know if we should change it. What do you think?

Reply:
OK, just leave it as it is.